materialize/resources/resource_source_kafka_test.go

```diff
@@ -22,27 +22,27 @@ func TestResourceSourceKafkaCreate(t *testing.T) {
 		"item_name":                  "item",
 		"kafka_connection":           "kafka_conn",
 		"topic":                      "topic",
-		"include_key":                "key",
+		"include_key":                true,
 		"include_headers":            true,
-		"include_partition":          "parition",
-		"include_offset":             "offset",
-		"include_timestamp":          "timestamp",
+		"include_partition":          true,
+		"include_offset":             true,
+		"include_timestamp":          true,
 		"format":                     "AVRO",
 		"key_format":                 "AVRO",
 		"envelope":                   "UPSERT",
 		"schema_registry_connection": "csr_conn",
 		"value_strategy":             "avro_key_fullname",
-		// "primary_key":                []interface{}{"key_1", "key_2", "key_3"},
-		// "start_offset":               []interface{}{1, 2, 3},
-		"start_timestamp": 100,
+		//"primary_key":                []interface{}{"key_1", "key_2", "key_3"},
+		//"start_offset":               []interface{}{1, 2, 3},
+		"start_timestamp": -1000,
```

Comment:
I'm actually not sure what the behaviour is if you specify a positive integer here. 🤔  Maybe it'll start from the beginning of the topic, rather than the end? Using a negative integer sounds less confusing and is likely the most common option for users. This is just a big ol' nit and I haven't double-checked the behaviour otherwise.

Reply:
👍 We can also include more validation in the parameter to ensure users only do negative

Reply:
Positive numbers are definitely allowed! Looking through our [test drive](https://github.com/MaterializeInc/materialize/blob/1481c8c4a5160845be2cd5e8e38c53e246e10a73/test/testdrive/kafka-time-offset.td#L71), it seems like this will literally match an existing Kafka timestamp (which...makes sense). Negative values are relative timestamps.